tests/lattice-controller.rs

```diff
@@ -0,0 +1,503 @@
+use std::collections::HashMap;
+use std::net::Ipv6Addr;
+use std::sync::Arc;
+
+use anyhow::{anyhow, Context, Result};
+use nkeys::KeyPair;
+use serde::Deserialize;
+use serde_json::json;
+use tokio::fs;
+use tokio::time::Duration;
+use tokio::try_join;
+use url::Url;
+use wascap::jwt;
+
+use wascap::wasm::extract_claims;
+use wasmcloud_control_interface::ClientBuilder;
+use wasmcloud_host::wasmbus::{Host, HostConfig};
+
+pub mod common;
+use common::free_port;
+
+use crate::common::nats::start_nats;
+use crate::common::{
+    assert_advertise_link, assert_start_actor, assert_start_provider, stop_server,
+};
+
+const TEST_LATTICE_PREFIX: &str = "test-lattice-controller";
+
+/// Test all functionality for the lattice-controller provider
+///
+/// - create_container
```

Comment:
Looks like a copy paste error from the blobstore tests?

Reply:
Yup, will fix the comment!

Reply:
Fixed comment here!